conduct-organizers.md

```diff
@@ -0,0 +1,66 @@
+# Triangle DevOps Code of Conduct - For Organizers
+
+This version is intended for internal use by the group organizers.
+
+## Warnings
+
+Any organizers can issue a verbal warning to a participant that
+their behavior violates the group's anti-harassment policy. Warnings should
+be reported to Mark Mzyk as soon as practical. The report should include:
+
+* Identifying information of the participant
+* The time you issued the warning
+* The behavior that was in violation
+* The approximate time of the behavior (if different than the time of warning)
+* The circumstances surrounding the incident
+* Your identity
+* Other people involved in the incident
+
+## Presentations
+
+Presentations or similar events should not be stopped for one-time gaffes or
+minor problems, although an organizer should speak to the presenter afterward.
+However, organizers should take immediate action to politely and calmly stop any
+presentation or event that repeatedly or seriously violates the anti-harassment
+policy. For example, simply say, "I'm sorry, this presentation cannot be
+continued at the present time" with no further explanation.
```

Comment:
Seems to me that it'd be all right to say this can't continue due to violating group policy. This seems like the bare minimum that would be needed to not cause everyone to wonder why you just stopped the presentation (although it would probably be obviously if this ever happened)